backend/director/db/sqlite/db.py

```
@@ -259,6 +261,87 @@ def delete_session(self, session_id: str) -> bool:
+                update_values.append(value)
+
+            update_fields.append("updated_at = ?")
+            update_values.append(int(time.time()))
+
+            update_values.extend([session_id])
+
+            query = f"""
+                UPDATE sessions 
+                SET {', '.join(update_fields)}
+                WHERE session_id = ?
+            """
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)